R/package.R

```diff
@@ -37,15 +39,27 @@
 #' })
 #'
 #' prefix
+#'
+#' # modify tolower to fail if given a non-character
+#' tolower <- argufy(base::tolower, x = ~ is.character)
```

Comment:
Maybe if you do `as.character` instead, then the example runs. Not a big deal, though.